imgcreate/editliveos.py

```diff
@@ -0,0 +1,2161 @@
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Library General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import division
+doc1 = '''
+              editliveos.py: [options] <LiveOS_source>
+
+              Edit a LiveOS image to merge a persistent overlay, insert files,
+              clone a customized instance, adjust the root or home filesystem
+              or overlay sizes, seclude private or user-specific files, rebuild
+              the image into a new, .iso image distribution file, and refresh
+              the source's persistent filesystem overlay.'''
+doc2 = '''
+ USAGE HELP   editliveos.py -h, -?, --help
+
+              LiveOS_source may be entered as "live" to edit or clone the
+              currently running LiveOS image.  An attached LiveOS loaded
+              device may be edited or cloned through its node id, such as
+              /dev/sdc1, or, if it is mounted, through its mount point path.
+              An .iso image file is addressed through its pathname, such as
+              /path/to/build.iso, or, if mounted, through the mount point
+              directory path.  A Live CD-ROM image is addressed through its
+              device node, such as /dev/sr0.  Even a directory containing a
+              LiveOS and iso/syslinux folders with the appropriate files can
+              be edited or cloned through that parent directory path.
+
+              The --clone option copies the home.img filesystem to the new
+              build .iso file, allowing user customizations to be replicated.
+              A version of livecd-iso-to-disk with the --copy-home loading
+              option may be used to propagate clones.
+
+              Other files and folders in the source device's outer file system
+              may be included in the new build.  Options are provided
+              to --include, --exclude, and --seclude files or folders for the
+              new build.  The new builds are branded to distinguish them with
+              the --name, --builder, and --releasefile options.
+
+              Storage space requirements for stageing the build files are
+              estimated by the script and compared to the space available in
+              the -t <TMPDIR> option (default=/var/tmp).
+
+              Invoke the --help option to learn about other optional features.
+              '''
+import os
+import sys
+import stat
+import glob
+import shutil
+import logging
+import tempfile
+import argparse
+import subprocess
+
+from fs import *
+from live import *
+from debug import *
+from errors import *
+from creator import *
+from kickstart import *
```

Comment:
Ideally, I'd prefer that all imports be namespaced.

That is, only use `import imgcreate.fs` rather than `from imgcreate.fs import *` and all function calls are called prefixed with the module name.

Since I have not yet changed all the rest of livecd-tools to do this, please at least change from `from fs import *` to `from imgcreate.fs import *`. That way it's easy to tell that these are internal modules and potential conflicts with system modules are avoided.